### PR TITLE
adding vignette

### PR DESCRIPTION
**Pull request by jwildfire**
fix #136

### CODE REVIEW

File: vignettes/my-vignette.Rmd [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+- The data must have one record per participant per timepoint per lab test. That is, the data should be long, not wide.
+- The data must have columns for: 
+  - Unique Subject Identifier (ID Column)
+  - Name of Measure or Test (Measure Column)
+  - Numeric finding or result (Value Column)
+  - Numeric upper limit of normal for the test
+  - Numeric lower limit of normal for the test
+  - Numeric study day
+- The following values must be present in the column corresponding to Measure or Test:
+  - Aminotransferase, alanine (ALT)
```
Comment (bzkrouse) [inline vignettes/my-vignette.Rmd]: Bullet points for ALT, AST, ALP, and TB need an extra indentation 
image

File: vignettes/my-vignette.Rmd [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+When you open the app, you are taken to the Data Tab with "Data Upload" and "Data Preview" panels. The preview for the pre-loaded "Example data" dataset is shown by default.
+
+<img src="https://user-images.githubusercontent.com/3680095/51489521-6b3f8600-1d5d-11e9-907a-1fce165edad9.png"  width="80%">
+
+To load your own data, simply click the browse button and select a `.csv` or `.sas7bdat` data set. Once the file is loaded, select it in the list at the bottom of the "Data Upload Panel". Once selected, the "Data Preview" panel will update automatically (along with the Settings and Chart tabs). 
+
+The charts in the safetyGraphics app are specifically designed for clinical trial safety monitoring, and require laboratory datasets that contain one row per participant per time point per measure. Data mappings for two common [CDISC](https://www.cdisc.org/) data standards - [SDTM](https://www.cdisc.org/standards/foundational/sdtm) and [ADaM](https://www.cdisc.org/standards/foundational/adam) - are pre-loaded in the application. As described below, the app can automatically generate charts for data sets using these standards; other data sets require some user configuration.
```
Comment (bzkrouse) [inline vignettes/my-vignette.Rmd]: ADaM links to a log-in page

File: vignettes/my-vignette.Rmd [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+```
+#Code to initialize shiny application
+devtools::install_github("ASA-DIA-InteractiveSafetyGraphics/safetyGraphics") 
+library("safetyGraphics") 
+safetyGraphicsApp()
+```
+
+### 2. Load Data
+
+Use the "Browse.." button on the data upload section of the data tab to load a non-standard data set. We'll use the `.csv` saved [here](https://github.com/ASA-DIA-InteractiveSafetyGraphics/safetyGraphics/raw/master/inst/eDISH_app/tests/partialSDTM.csv), but the process is similar for other data sets. Notice that once the data is loaded, the app will detect whether the data matches one of those pre-loaded standards, and a note is added to indicate whether a match is found. Our sample data is a partial match for the SDTM standard. Once you select the newly loaded data set, the app should look like the screen capture below. Note the red x's in the toolbar indicating that user customization is needed.
```
Comment (bzkrouse) [inline vignettes/my-vignette.Rmd]: csv link - page not found

File: vignettes/my-vignette.Rmd [hunk 4]
```diff
@@ -0,0 +1,216 @@
+---
+title: "safetyGraphics Shiny App - User Guide"
+author: "Jeremy Wildfire"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
```
Comment (bzkrouse) [inline vignettes/my-vignette.Rmd]: "Vignette Title" is where we can specify a name for this vignette.  It's what will appear on the CRAN page linking to the vignette.

File: vignettes/my-vignette.Rmd (hunk 1 above)
Comment (jwildfire) [reply vignettes/my-vignette.Rmd]: good catch. will fix.

**Pull request merged**